main/FatesHistoryInterfaceMod.F90

```diff
@@ -3541,7 +3469,7 @@ subroutine update_history_dyn2(this,nc,nsites,sites,bc_in)
                       leaf_m   = ccohort%prt%GetState(leaf_organ, element_list(el))
                       fnrt_m   = ccohort%prt%GetState(fnrt_organ, element_list(el))
                       store_m  = ccohort%prt%GetState(store_organ, element_list(el))
-                      repro_m  = ccohort%prt%GetState(repro_organ, element_list(el))
+                      repro_m  = ccohort%prt%GetState(repro_organ, element_list(el))  ! TODO: Unused?
```

Comment:
Note TODO.

Reply:
- [ ] Add comment that this should be zero

Reply:
Resolved as of 2242bd1d.